articles/service-fabric/service-fabric-backup-restore-service-ondemand-backup.md

```diff
@@ -87,9 +87,12 @@ Use the [BackupPartition](/rest/api/servicefabric/sfclient-api-backuppartition)
 
 ```powershell
 $StorageInfo = @{
-    ConnectionString = 'DefaultEndpointsProtocol=https;AccountName=<account-name>;AccountKey=<account-key>;EndpointSuffix=core.windows.net'
-    ContainerName = 'backup-container'
-    StorageKind = 'AzureBlobStore'
+    StorageKind = "ManagedIdentityAzureBlobStore"
+    FriendlyName = "AzureMI_storagesample"
```

Comment:
Have we checked if Friendly Name is a mandatory parameter. seems superficial to me.

Reply:
it is and optional param